assets/materials/shaders/CMakeLists.txt

```diff
@@ -37,3 +37,11 @@ generate_rules_for_shader("shader_mtl_pbr"
     SOURCE "${PPX_DIR}/assets/materials/shaders/PBR.hlsl"
     INCLUDES ${INCLUDE_FILES}
     STAGES "ps")
+
+generate_rules_for_shader("shader_mtl_env_draw_vertex_shader"
```

Comment:
This can be a single rule with `STAGES "vs" "ps"`.

Reply:
Fixed. Thanks didn't know this was a thing!